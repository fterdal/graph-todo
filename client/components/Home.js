import React from 'react';

export const Home = () => {
  return (
    <div className="container">
      <h1>Home Page</h1>
    </div>
  )
}

export default Home;
