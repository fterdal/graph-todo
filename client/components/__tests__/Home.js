import React from 'react';
import renderer from 'react-test-renderer';
import { Home } from '../Home';

test('Home renders a simple div', () => {
  const home = renderer.create(
    <Home />,
  );
  let homeTree = home.toJSON();
  expect(homeTree).toMatchSnapshot();
});
