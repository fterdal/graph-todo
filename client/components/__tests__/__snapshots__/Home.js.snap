// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a simple div 1`] = `
<div
  className="container"
>
  <h1>
    Home Page
  </h1>
</div>
`;
